rpms/base.pan

```diff
@@ -58,6 +65,10 @@ variable OS_CORE_RDMA_ENABLED ?= false;
   # ncm-spma will keep trying removing all kernels and only the running one will be kept.
   pkg_repl('kernel');
 
+  # chkconfig: required by several Quattor components
```

Comment:
Hmm, we should have a list of which require it.

Reply:
I think it is misworded. Components should be interpreted in the generic meaning, part of Quattor. I don't think any configuration module still requires it but we should check in grid components may be. As for the `chkconfig` usage in various templates, I think I cleaned most of the template library now (but not sure if I upstreamed everything). At least EL9 OS templates should be free of it. For the short term I suggest to rephrase the comment.